session/bootstrap.go

```diff
@@ -1546,6 +1549,13 @@ func upgradeToVer73(s Session, ver int64) {
 	doReentrantDDL(s, CreateCapturePlanBaselinesBlacklist)
 }
 
+func upgradeToVer74(s Session, ver int64) {
+	if ver >= version74 {
+		return
+	}
+	mustExecute(s, fmt.Sprintf("UPDATE mysql.global_variables SET VARIABLE_VALUE='%[1]v' WHERE VARIABLE_NAME = 'tidb_stmt_summary_max_stmt_count' AND CAST(VARIABLE_VALUE AS SIGNED) < %[1]v", config.GetGlobalConfig().StmtSummary.MaxStmtCount))
```

Comment:
I don't think so. What if the user has a custom value, say, 500? It's weird to change that.

Reply:
Yes, but It's not bad that change from 500 to 3000?

Reply:
Is it acceptable to only upgrades from `200`?

Reply:
I still prefer to enlarge this value when the value is small than 200.  With the bigger capacity of `statements_summary`, it is much more useful in troubleshooting.

Reply:
ok

Reply:
@xhebox PTAL

Reply:
I remembered you have replied: I still prefer to enlarge the value. Whatever, this has been resolved.